### ISSUE TITLE
Q-numeric on ios : value partially hidden

### ISSUE THREAD

**Issue by caex** [quasarframework/quasar#290]
Hi

using q-numeric on ios (latest), the value is partially hidden : 

![image](https://cloud.githubusercontent.com/assets/4222866/21554336/2598e0aa-ce0f-11e6-93ac-7b04cbe5fcec.png)

The simplest workaround I've found is :
````
.q-numeric input {
  padding-left: 0;
  padding-right: 0;
}
````
![image](https://cloud.githubusercontent.com/assets/4222866/21554356/56ff5cd2-ce0f-11e6-8347-f8877de2074b.png)

The 0 left/right padding is default on desktop Chrome : 
![image](https://cloud.githubusercontent.com/assets/4222866/21554394/9712874a-ce0f-11e6-90b1-8ef0a49de0eb.png)
So I guess this may have something to do with ios' default style sheet, but not having a Mac I cannot check.

Cheers

**Comment by rstoenescu** [comment]
Confirmed. Working on a fix. Thanks for reporting this!

**Comment by rstoenescu** [comment]
You'll be credited with the fix as it is the right one to make. Thanks for reporting and also investigating!

**Comment by rstoenescu** [comment]
Fix available in Quasar edge and future v0.12
